plugins/scm.js

```diff
@@ -29,6 +29,12 @@ const ADD_WEBHOOK = Joi.object().keys({
     scmContext
 }).required();
 
+const PARENT_CONFIG = Joi.object().keys({
+    branch: Joi.string().required(),
+    host: Joi.string().required(),
+    org: Joi.string().required(),
+    repo: Joi.string().required()
```

Comment:
You probably also need `sha` here for restarting past build from the child pipeline.

Reply:
Good point! Will add